src/index.html

```diff
@@ -6,21 +6,428 @@
+    
+                                       </div> 
+                                    </div>
+                                    <span class="detail_title">Area</span> 
+                                </div>
+                            
+                        </div>
+                        <div class="estate_price_status">
+                            <div class="estate_price">
+                                $630,000
+                            </div>
+                            <div class="estate_status">
+                                FOR RENT
+                            </div>
+                        </div>
+                    </div>
+                </li>
+                
+                
+            </ul>
+            <div class="option_buttons">
+                <button class="option_button">VIEW MORE SALE</button>
+                <button class="option_button">VIEW MORE RENT</button>
+            </div>
+        </div>
+        <footer class="general_footer">
+            <div class="footer_element  nav_contact">
+                <ul class="nav_menu2">
+                    <li class="nav_menu2_element">Home</li>
+                    <li class="nav_menu2_element">Buy</li>
+                    <li class="nav_menu2_element">Rent</li>
+                    <li class="nav_menu2_element">About Us</li>
+                    <li class="nav_menu2_element">News</li>
+                    <li class="nav_menu2_element">Contact</li>
+
+                </ul>
+                <div class="nav_contact_separator"></div>
+                <ul class="address_list">
+                    <li class="address_list_element">Easy House Renting Agency</li>
+                    <li class="address_list_element">Transaction / Gestion / Location</li>
+                    <li class="address_list_element">306 Rivera St, WellCity, BOLIVIA</li>
+                    <li class="address_list_element">+2135 89 6558 401 235</li>
+                    <li class="address_list_element">info@easyhouse.bl</li>
+                </ul>
+                <div class="nav_contact_separator"></div>
+                <div class="social_media_list">
+                    <img class="social_media_element" src="./assets/facebook-icon.svg" alt="">
+                    <img class="social_media_element" src="./assets/instagram-icon.svg" alt="">
+                    <img class="social_media_element" src="./assets/twitter-icon.svg" alt="">
+                </div>
+            </div>
+            <div class=" footer_element copyright">
+                <div class="copyright_text">copyright  @ 2023 </div>
+                <div class="separator"></div>
+                <div class="copyright_text">Link privacy</div>
+                <div class="separator"></div>
+                <div class="copyright_text">Website by Nela&Koami </div>
+                <div class="separator"></div>
+                <div class="copyright_text">TVA: 125463578900 </div>
+
+            </div>
```

Comment:
<img width="1349" alt="footer" src="https://github.com/Nelitaa/EasyHouse/assets/98427585/b8b77f9c-6631-47c0-b85d-e3433a1a1087">
To align the content more closely with the original design, consider adding some margin to this section. This could help achieve the desired visual layout.